src/icebreaker/five_figures.py

```diff
@@ -11,31 +11,32 @@
 def main(mic_path):
 	orig_stdout = sys.stdout
 	indir = mic_path
-	f = open('five_figs_test.csv','w')
-	sys.stdout = f
-	files = []
-	for filename in os.listdir(indir):
-		if (filename.endswith(".mrc")):
-			finalpath = os.path.join(indir,filename)
-			files.append(finalpath)
-		else:
-			continue
-#files = files.sort()
-	files = sorted(files)
-	vector_data = []
-	r=10000
-
-	print('path,','min,','q1,','q2=median,','q3,','max')
-	for img_path in files:
-		with mrcfile.open(img_path, 'r', permissive=True) as mrc:
-			img = mrc.data
-			if(np.isnan(np.sum(img))):
-				continue
+	with open('five_figs_test.csv','w') as f:
+		files = []
+		for filename in os.listdir(indir):
+			if (filename.endswith(".mrc")):
+				finalpath = os.path.join(indir, filename)
+				files.append(finalpath)
 			else:
-				print(img_path[:-4]+',',str(int(np.min(img)*r))+',',str(int(np.quantile(img,0.25)*r))+',',str(int(np.median(img)*r))+',',str(int(np.quantile(img,0.75)*r))+',',int(np.max(img)*r))
+				continue
+		# files = files.sort()
+		files = sorted(files)
+		vector_data = []
+		r = 10000
 
-	sys.stdout = orig_stdout
+		f.write('path,min,q1,q2=median,q3,max')
+		for img_path in files:
+			with mrcfile.open(img_path, 'r', permissive=True) as mrc:
+				img = mrc.data
+				if (np.isnan(np.sum(img))):
+					continue
+				else:
```

Comment:
```suggestion
```suggestion
		for img_path in sorted(files):
			with mrcfile.open(img_path, 'r', permissive=True) as mrc:
				img = mrc.data
			if not np.isnan(np.sum(img)):
```
simplify control flow, no need to have an `if: continue` if the path is otherwise empty anyway, close the .mrc file once it's no longer needed.